cmake/modules/FindRuby.cmake

```diff
@@ -15,7 +15,7 @@ if(RUBY_LIBRARY AND RUBY_INCLUDE_PATH)
    set(RUBY_FIND_QUIETLY TRUE)
 endif (RUBY_LIBRARY AND RUBY_INCLUDE_PATH)
 
-FIND_PROGRAM(RUBY_EXECUTABLE NAMES ruby ruby1.9 ruby19 ruby1.9.1 ruby2.1)
+FIND_PROGRAM(RUBY_EXECUTABLE NAMES ruby ruby2.4 ruby.ruby2.4 ruy2.5 ruby.ruby2.5)
```

Comment:
this should be needed. We should search for only ruby. Version specific ones is problem as have version specific RbConfig which we do not want. We want only system one.